examples/c++/ntt/compile.sh

```diff
@@ -3,9 +3,12 @@
 # Exit immediately on error
 set -e
 
-rm -rf build
 mkdir -p build
-cmake -S . -B build
-cmake --build build
 
+# Configure and build Icicle
```

Comment:
Maybe use another script that is building icicle and exports the build/include dir and source it from here

Reply:
Isn't this way more realistic right now since each integration needs to build from source for their own needs? I don't imagine them having two separate build scripts

Reply:
I think there are two options: (1) build (or install with the include dir) in some directory and link to that which is how it is now or (2) do it directly via cmake (FetchContent).